lib/Command/ExpirePassword.php

```diff
@@ -124,7 +124,6 @@ protected function configure() {
 	 *
 	 * @param InputInterface $input
 	 * @param OutputInterface $output
-	 * @return int
```

Comment:
execute() should return an `int` status. It already has `return 1` `return 2`  for some error conditions.
IMO there should be a `return 0` at the end of the function.